processor/k8sattributesprocessor/doc.go

```diff
@@ -117,12 +141,22 @@
 // 1. Use the downward API to inject the node name as an environment variable.
 // Add the following snippet under the pod env section of the OpenTelemetry container.
 //
-//    env:
-//    - name: KUBE_NODE_NAME
-//      valueFrom:
-//  	  fieldRef:
-//  	    apiVersion: v1
-//  	    fieldPath: spec.nodeName
+//
+//    spec:
+//      containers:
+//      - env:
+//        - name: KUBE_NODE_NAME
+//          valueFrom:
+//            fieldRef:
+//             apiVersion: v1
+//              fieldPath: spec.nodeName
+//        - name: POD_IP
+//          valueFrom:
+//            fieldRef:
+//              apiVersion: v1
+//              fieldPath: status.podIP
+//        - name: OTEL_RESOURCE_ATTRIBUTES
+//          value: k8s.pod.ip=$(POD_IP)
```

Comment:
`k8s.pod.ip` is supposed to be IP of a pod emitting data, not the collector itself. It's usually taken from the context of tracing data.

So this is correct configuration part that can be set on **instrumented pod**, not on the cllector